pkg/declarativeconfig/permission_set.go

```diff
@@ -0,0 +1,33 @@
+package declarativeconfig
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"gopkg.in/yaml.v3"
+)
+
+type PermissionSet struct {
+	Name             string              `yaml:"name,omitempty"`
+	Description      string              ` yaml:"description,omitempty"`
+	ResourceToAccess map[string]DCAccess `yaml:"resource_to_access,omitempty"`
+}
+
+type DCAccess storage.Access
```

Comment:
I think in the scope of this file, you can omit the prefix of `DC` and simply put `Access` here?